public/styles/components/_minimap.css

```
@@ -0,0 +1,179 @@
+    transform 0.3s ease;
+}
+
+/* Minimap Canvas */
+.minimap-canvas {
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+
+  /* High DPI support */
+  image-rendering: -webkit-optimize-contrast;
+  image-rendering: -moz-crisp-edges;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

image-rendering is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)